### PR TITLE
adding shadie package

### PR DESCRIPTION
**Pull request by elissasoroj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by elissasoroj** [comment]
I confirm that I am willing to maintain this package

**Comment by elissasoroj** [comment]
@conda-forge/help-python, ready for review!

File: recipes/shadie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/elissasoroj/shadie/archive/v{{ version }}.tar.gz
+  sha256: 9d87d9c6dc14099721ab6482ab74793e6673a870d8fdd09644e072b10edabc1a
+
+build:
+  noarch: python
```
Comment (xhochy) [inline recipes/shadie/meta.yaml:13]: If your package contains compiled code, it cannot be noarch.
```suggestion
```

File: recipes/shadie/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
```
Comment (xhochy) [inline recipes/shadie/meta.yaml:20]:
```suggestion
```

File: recipes/shadie/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python >=3.10
```
Comment (xhochy) [inline recipes/shadie/meta.yaml:24]:
```suggestion
    - python
```

File: recipes/shadie/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  run:
+    - altair
+    - loguru
+    - numpy
+    - pandas
+    - pyslim
+    - python >=3.6
```
Comment (xhochy) [inline recipes/shadie/meta.yaml:31]:
```suggestion
    - python
```

File: recipes/shadie/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    - scipy
+    - toyplot
+    - toytree
+    - tskit
+
+test:
```
Comment (xhochy) [inline recipes/shadie/meta.yaml:37]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/shadie/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    - pip
+    - python >=3.7
+
+  run:
+    - python >=3.7
+    - toytree >=3.0.0
+    - altair >=5.2.0
+    - pyslim >=1.0.3
+    - slim >=4.1.0
+    - bioconda::python-newick >=1.3.0 #for msprime
```
Comment (xhochy) [inline recipes/shadie/meta.yaml:28]: It is not OK to depend on bioconda. Everything needs to be in conda-forge.

**Review by xhochy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**